DFE-Digital/ecf-engage-and-learn
# Show year title for mentor in breadcrumbs

## Ticket and context

Ticket: [551](https://dfedigital.atlassian.net/jira/software/projects/CPDEL/boards/89?assignee=5f3543bb3e9e2e004db3189d&selectedIssue=CPDEL-551)

## Code review

### Is there anything weird that code reviewer should know?
Nope, everything is straightforward.

### Review Checks

## Product review

ECT Year 1 breadcrumb
<img width="558" alt="Screenshot 2021-06-10 at 09 41 34" src="https://user-images.githubusercontent.com/69353998/121494428-a6938b80-c9d0-11eb-89ca-c1c9e457142f.png">
ECT Year 2 breadcrumb
<img width="567" alt="Screenshot 2021-06-10 at 09 41 28" src="https://user-images.githubusercontent.com/69353998/121494461-abf0d600-c9d0-11eb-9e8a-2699802b22cb.png">
Mentor Year 1 breadcrumb
<img width="543" alt="Screenshot 2021-06-10 at 09 40 58" src="https://user-images.githubusercontent.com/69353998/121494476-af845d00-c9d0-11eb-94df-8860bd6a2df9.png">
Mentor Year 2 breadcrumb
<img width="553" alt="Screenshot 2021-06-10 at 09 40 51" src="https://user-images.githubusercontent.com/69353998/121494488-b1e6b700-c9d0-11eb-960f-a8d595efb9ef.png">

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Login as admin, choose a cip.
3. Select edit course year, note the ect title and mentor title, change it if you like. 
4. Select Year 2, do as you've done in step 3.
5. Logout
6. Sign in as a mentor for the provider you edited, go to the year page and you should see the mentor year title in the breadcrumb.
7. Logout
8. Sign in an ECT for the provider you edited, go to the year page and you should see the ECT year title in the breadcrumb
